test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -3289,6 +3289,7 @@ static int test_query_functions(const char *shaper_name,
 	expected_pkt_cnt  = num_pkts - 2;
 	expected_byte_cnt = expected_pkt_cnt * PKT_BUF_SIZE;
 
+	memset(&query_info, 0, sizeof(query_info));
 	rc = odp_tm_queue_query(tm_queue,
```

Comment:
This seems more like a bug in the test itself. If `odp_tm_queue_query()` fails the test still tries to read `query_info` contents. Does replacing the following CU_ASSERT with CU_ASSERT_FATAL fix the warning? The same issue is repeated with each query call in `test_query_functions`.

Reply:
Nope, doesn't help.

Reply:
If PR #987 gets merged, CU_ASSERT_FATAL should get rid of the warning here.

However, there is an even bigger bug in the test: It does not actually test if the latter query functions write the query info at all. If the first query succeeds, query_info already contains the right content even if the implementation of the later called query functions were simply {return 0;}. The test would not notice anything wrong. So clearing the query info content is needed because of that and then the CU_ASSERT versus CU_ASSERT_FATAL issue becomes moot.